concordia/static/scss/base.scss

```diff
@@ -363,7 +363,7 @@ ul.nav-secondary {
 .btn:disabled {
     opacity: 0.3 !important;
 }
-
+/*
```

Comment:
We prefer not to commit commented-out code blocks since if we need to pull something back we can get it from the Git history